.circleci/config.yml

```
@@ -18,8 +28,36 @@ jobs:
     - store_test_results:
         path: /tmp/results
 
+  publish-github-release:
+    docker:
+      - image: cibuilds/github:0.12
+    steps:
+      - checkout
+
+      - run:
+          name: "Package Helm Chart"
+          command: |
+            ./ci-scripts/package.sh ${CIRCLE_PROJECT_REPONAME} ${CIRCLE_TAG}
+
+      - run:
+          name: "Publish Release on GitHub"
+          command: |
+            ghr -t ${PERSONAL_ACCESS_TOKEN} -u giantswarm -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} ${CIRCLE_TAG} "${CIRCLE_PROJECT_REPONAME}-chart-${CIRCLE_TAG}.tgz"
+
 workflows:
   version: 2
   build-deploy:
     jobs:
-      - build
+      - lint
+      - build:
+          filters:
+            tags:
+              only: /^\d+\.\d+\.\d+$/
```

Let's pease use tag in format `v1.2.3`. This is not only kind of standard, but also works well with go modules.

```suggestion
              only: /v^\d+\.\d+\.\d+$/
```

---

@puja108 and @giantswarm/sig-releng do we also want to have our Chart versions prefixed with `v` then, or only the tags?

---

hmm, good call, now that you say it upstream standard is not having a v there, so might be nicer to not have it, just cause we do that internally a lot and it's easier with modules doesn't mean it needs to be in the chart and with that the URL of the tgz

---

For the `v` prefix I'd prefer it if we were consistent. Having to remember to use it sometimes and not others is annoying and error prone.

---

It doesn't need to be in the chart, but I insist to have git tags in `v1.2.3` format. Otherwise we'll have to make tooling aware of both.

---

I'd argue that if both things are following standards it's even easier to remember.

---

you're right, when I look at most tags/releases on github in kubernetes world they do carry `v`, and indeed with chart versions internally that is not the case, so that is the only expection here

TLDR standards are v everywhere but in the version field of a yaml (kinda :wink: )

---

I think the regex here should be `/^v\d+\.\d+\.\d+$/` (`v` after `^`)

---

Thanks @TheoBrigitte. Updated.

---

Yeah, let's go with that for now. Everyone ok with that?

---

Cool. Sorted and updated. PTAL @kopiczko